lib/sharezone_plus/sharezone_plus_page_ui/lib/src/sharezone_plus_faq.dart

```diff
@@ -13,23 +13,32 @@ import 'package:url_launcher/link.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 class SharezonePlusFaq extends StatelessWidget {
-  const SharezonePlusFaq({super.key});
+  const SharezonePlusFaq({
+    super.key,
+    this.showContentCreatorQuestion = false,
```

Comment:
Not showing the question on the "Eltern bezahlen lassen" site since has no value for parents.